sklearn/multioutput.py

```diff
@@ -124,10 +125,12 @@ def partial_fit(self, X, y, classes=None, sample_weight=None):
                 "multi-output regression but has only one."
             )
 
-        if sample_weight is not None and not has_fit_parameter(
-            self.estimator, "sample_weight"
-        ):
-            raise ValueError("Underlying estimator does not support sample weights.")
+        routed_params = process_routing(
```

Comment:
**Not actionable:** I think the only way for a third party meta-estimator developer to support multiple versions of scikit-learn is to vendor `_metadata_requests.py`. Is this what you envisioned?